### PR TITLE
Update how we work and feature issue template

### PR DESCRIPTION
**Pull request by elainekamlley**
Edited the how we work doc to reflect our changes to the process. Also edited the feature issue template to better align us for backlog grooming, sprint planning and testing.

### CODE REVIEW

File: docs/how-we-work.md [hunk 1]
```diff
@@ -6,31 +6,39 @@ Our basic principles are those familiar to anybody who has contributed to a prom
 
 -   We produce open source software, created and maintained in repositories where it may be inspected by the public who places their trust in it and copied for use by other agencies or institutions.
 -   We adhere to the basic practices of agile software development, using the Scrum development framework.
--   We practice human-centered design. Everything that we produce is highly accessible, per [WCAG 2.1](https://www.w3.org/TR/WCAG21/).
+-   We practice human-centered design. Everything that we produce is highly accessible, per [WCAG 2.0](https://www.w3.org/TR/WCAG20/).
```
Comment (acdha) [inline docs/how-we-work.md]: WCAG 2.1 has a number of improvements over 2.0 and is backwards-compatible. If we're intentionally not adopting it we should document why.

https://www.w3.org/WAI/standards-guidelines/wcag/#versions
> Content that conforms to WCAG 2.1 also conforms to WCAG 2.0. (This is often called “backwards compatible”.) A website that meets WCAG 2.1 should meet the requirements of policies that reference WCAG 2.0.
>
> To put it another way: If you want to meet both WCAG 2.0 and WCAG 2.1, you can use the 2.1 resources and you don’t need to bother looking at 2.0.
> 
> WCAG 2.0 and WCAG 2.1 are both existing standards. WCAG 2.1 does not deprecate or supersede WCAG 2.0. W3C encourages you to use the most recent version of WCAG when developing or updating content or accessibility policies.

File: docs/how-we-work.md [hunk 2]
```diff
@@ -79,15 +88,15 @@ Testers will identify high, medium, low. Here are the criteria for each of the l
...
 
--   Open a new BUG ticket if it is a high
--   Open a new FEATURE ticket even if enhancement if it is a low
--   Link to testing plan that is in the wiki
+-   For a BUG, comment and @ the developer in the feature ticket. The ticket number will added to the testing plan.
+-   If it is an enhancement, open a new FEATURE ticket. This ticket will be added to the backlog and up for priortization in alignment meeting and backlog grooming
+-   Link to testing plan that is in the wiki, to the issue
 
-If a Bug ticket is open be prepared to be available to on Thursday and Friday to respond to developers until result is what is desired. If a high priority bug ticket is created, it can’t be shipped until tester closes ticket
+If a BUG needs to be addressed, tester should expect to be available to on Thursday and Friday to respond to developers questions until acceptance criteria is met.
```
Comment (acdha) [inline docs/how-we-work.md]:
```suggestion
If a BUG needs to be addressed, tester should expect to be available on Thursday and Friday to respond to questions and retest until acceptance criteria is met. 
```

File: docs/how-we-work.md (hunk 1 above)
Comment (elainekamlley) [reply docs/how-we-work.md]: Something to catch about with Zuhair when we review our own accessibility policy. When we talked, the library officially meets 2.0 and has yet to make the policy change to meet 2.1.

**Comment by elainekamlley** [comment]
Not sure why its failing, any thoughts @rstorey or @acdha

File: docs/how-we-work.md (hunk 1 above)
Comment (acdha) [reply docs/how-we-work.md]: Since 2.1 is a superset of 2.0 I would look forward and let the main policy catch up.

**Comment by acdha** [comment]
@elainekamlley There's some Markdown syntax cleanup which Prettier flagged:

https://travis-ci.org/LibraryOfCongress/concordia/jobs/477882145#L683-L688

**Comment by elainekamlley** [comment]
Ah, I'll clean that up.

**Pull request merged**